### PR TITLE
Expand the `./resources` README

### PR DESCRIPTION
**Pull request by rocco8773**
...

### CODE REVIEW

File: resources/README.md [hunk 1]
```diff
@@ -1,4 +1,86 @@
...
+installing Python 3.10.  A Python distribution can be
+[downloaded][python-download] directly from python.org and installed.
+
+### [Anaconda]
+
+[Anaconda] is similar to the Python Package Index (PyPI), in that it
+is a package distribution hub.  Unlike PyPI, packages listed on
+Anaconda are not restricted to just Python packages.
+
+Anaconda has its own package installer `conda`, similar to PyPI's
```
Comment (namurphy) [inline resources/README.md:45]:
```suggestion
Anaconda has its own package installer called `conda`, similar to PyPI's
```

**Comment by rocco8773** [comment]
@namurphy I left the "Event Day Resources" section empty, but will add to it in follow up PRs as those resource materials are added to the repo.

File: resources/README.md [hunk 2]
```diff
@@ -1,4 +1,122 @@
...
+[VSCode]: https://code.visualstudio.com/
+
+
 # Plasma Hack Week Resources
 
-This director is intended to contain resources like pre-event materials,
-notebooks, tutorials, etc. to help with the Hack Week event.
+This directory contains resources pertinent to the 2022 Plasma Hack Week.
+Added resources include, but are not limit to, pre-event background
+materials, demo jupyter notebooks, tutorials, etc.
```
Comment (namurphy) [inline resources/README.md:18]:
```suggestion
materials, demo Jupyter notebooks, tutorials, etc.
```

**Review by namurphy — APPROVED** [review]
Looks good!  One optional unimportant suggestion.

**Pull request merged**